cmd/policy_webhook/main.go

```diff
@@ -64,6 +65,10 @@ var (
 	//    ./config/501-policy-webhook-configurations.yaml
 	//    https://github.com/sigstore/helm-charts/blob/main/charts/policy-controller/templates/policy-webhook/policy_webhook_configurations.yaml
 	validatingWebhookName = flag.String("validating-webhook-name", "validating.clusterimagepolicy.sigstore.dev", "The name of the validating webhook configuration as well as the webhook name that is automatically configured, if exists, with different rules and client settings setting how the admission requests to be dispatched to policy-webhook.")
+
+	// policyResyncPeriod holds the interval which ClusterImagePolicies will resync
+	// This is essential for triggering a reconcile update for potentially stale KMS authorities.
+	policyResyncPeriod = flag.String("policy-resync-period", "10h", "The resync period for ClusterImagePolicies. The default is 10h.")
```

Comment:
Suggestion: I'll add a field to the configMap to set this value and change it without having to restart the server. We have added several new configuration settings to that config map.

Reply:
See the earlier discussion where putting it there and making it change without having to restart the server is fairly complex, unless I'm unaware of something easier to actually do this.

Reply:
That could be done by making some changes to knative Impl controller so the resync period could be refreshed whenever the configmap changes.
Perhaps it is too much travel for the benefit of adding that support.